tests/src/main/java/net/neoforged/neoforge/debug/world/WorldgenRegistryDesyncTest.java

```diff
@@ -13,16 +15,14 @@
 import net.neoforged.fml.common.Mod;
 import net.neoforged.fml.javafmlmod.FMLJavaModLoadingContext;
 import net.neoforged.neoforge.registries.DeferredRegister;
-import net.neoforged.neoforge.registries.ForgeRegistries;
-import net.neoforged.neoforge.registries.RegistryObject;
 
 @Mod("worldgen_registry_desync_test")
 public class WorldgenRegistryDesyncTest {
 
-    public static final DeferredRegister<Feature<?>> FEATURES = DeferredRegister.create(ForgeRegistries.FEATURES, "worldgen_registry_desync_test");
-    public static final RegistryObject<Feature<NoneFeatureConfiguration>> dungeon = FEATURES.register("dungeon", () -> new MonsterRoomFeature(NoneFeatureConfiguration.CODEC));
-    public static final DeferredRegister<Biome> BIOMES = DeferredRegister.create(ForgeRegistries.BIOMES, "worldgen_registry_desync_test");
-    //TODO: public static final RegistryObject<Biome> biome = BIOMES.register("biome", VanillaBiomes::theVoidBiome);
+    public static final DeferredRegister<Feature<?>> FEATURES = DeferredRegister.create(Registries.FEATURE, "worldgen_registry_desync_test");
+    public static final Holder<Feature<?>> dungeon = FEATURES.register("dungeon", () -> new MonsterRoomFeature(NoneFeatureConfiguration.CODEC));
+    public static final DeferredRegister<Biome> BIOMES = DeferredRegister.create(Registries.BIOME, "worldgen_registry_desync_test");
+    //TODO: public static final Holder<Biome> biome = BIOMES.register("biome", VanillaBiomes::theVoidBiome);
```

Comment:
Non-datapack registration of biomes is not a thing anymore, so this might as well be removed if we have to touch it anyway.